django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -19,6 +19,7 @@
 
 .selector-chosen select {
     border-top: none;
+    height: 20.8em;
```

Comment:
To be honest, I've chosen 20.8em experimentally because with this height, they seem vertically aligned. There is no calculation behind it.